### ISSUE TITLE
Segfault with `startup_commands toggle_presentation_mode` on Linux

### ISSUE THREAD

**Issue by rkern** [ahrm/sioyek#310]
I'm using the 1.4.0 `sioyek-release-linux.zip` on Ubuntu 22.04. If I add `startup_commands toggle_presentation_mode` to my `~/.config/sioyek/prefs_user.config` file, then I get a segfault on startup. Other preferences work fine, as do other startup commands like `fit_page_height`.

```
❯ cat ~/.config/sioyek/prefs_user.config
# Semicolon-separated list of command to execute upon sioyek startup
startup_commands    toggle_presentation_mode

❯ ~/.local/bin/Sioyek-x86_64.AppImage
default_config_path: /tmp/.mount_Sioyek2K7A47/usr/bin/prefs.config
default_keys_path: /tmp/.mount_Sioyek2K7A47/usr/bin/keys.config
user_config_path: [ 0 ] /usr/share/kubuntu-default-settings/kf5-settings/Sioyek-x86_64.AppImage/prefs_user.config
user_config_path: [ 1 ] /etc/xdg/Sioyek-x86_64.AppImage/prefs_user.config
user_config_path: [ 2 ] /etc/xdg/xdg-plasma/Sioyek-x86_64.AppImage/prefs_user.config
user_config_path: [ 3 ] /home/rkern/.config/kdedefaults/Sioyek-x86_64.AppImage/prefs_user.config
user_config_path: [ 4 ] /home/rkern/.config/Sioyek-x86_64.AppImage/prefs_user.config
user_config_path: [ 5 ] /home/rkern/.local/share/Sioyek/prefs_user.config
user_config_path: [ 6 ] /home/rkern/.config/sioyek/prefs_user.config
user_keys_path: [ 0 ] /usr/share/kubuntu-default-settings/kf5-settings/Sioyek-x86_64.AppImage/keys_user.config
user_keys_path: [ 1 ] /etc/xdg/Sioyek-x86_64.AppImage/keys_user.config
user_keys_path: [ 2 ] /etc/xdg/xdg-plasma/Sioyek-x86_64.AppImage/keys_user.config
user_keys_path: [ 3 ] /home/rkern/.config/kdedefaults/Sioyek-x86_64.AppImage/keys_user.config
user_keys_path: [ 4 ] /home/rkern/.config/Sioyek-x86_64.AppImage/keys_user.config
user_keys_path: [ 5 ] /home/rkern/.local/share/Sioyek/keys_user.config
user_keys_path: [ 6 ] /home/rkern/.config/sioyek/keys_user.config
database_file_path: /home/rkern/.local/share/Sioyek/test.db
local_database_file_path: /home/rkern/.local/share/Sioyek/local.db
global_database_file_path: /home/rkern/.local/share/Sioyek/shared.db
tutorial_path: /home/rkern/.local/share/Sioyek/tutorial.pdf
last_opened_file_address_path: /home/rkern/.local/share/Sioyek/last_document_path.txt
shader_path: /tmp/.mount_Sioyek2K7A47/usr/bin/shaders
Creating shared memory block...
Shared memory created: this is the primary application.
Starting IPC server...
IPC server started.
[1]    1923718 segmentation fault (core dumped)  ~/.local/bin/Sioyek-x86_64.AppImage
```

**Comment by ahrm** [comment]
Should be fixed in 9615820064606887c43c77bad2b001ddabb064b2.

**Comment by rkern** [comment]
Such service! Thank you!
